applications/serial_lte_modem/doc/MQTT_AT_commands.rst

```diff
@@ -0,0 +1,254 @@
+.. _SLM_AT_MQTT:
+
+MQTT AT Commands
+****************
+
+The following commands list contains the AT commands used to operate the MQTT client:
+
+* `MQTT Connect #XMQTTCON`_
+* `MQTT Subscribe #XMQTTSUB`_
+* `MQTT Publish #XMQTTPUB`_
+
+----
+
+MQTT Connect #XMQTTCON
+======================
+
+The ``#XMQTTCON`` command allows to connect to and disconnect from the MQTT broker.
+
+Set command
+-----------
+
+The set command allows to control to connect to and disconnect from the MQTT broker.
+
+Syntax
+~~~~~~
+
+::
+
+   AT#XMQTTCON=<op>[,<client_id>,<url>,<port>[,<sec_tag>]]
+
+
+* The ``<op>`` parameter is an integer and can accept one the following values:
+
+  * ``0`` - Connect to the MQTT broker.
+  * ``1`` - Disconnect from the MQTT broker.
+
+* The ``<client_id>`` parameter is a string and it indicates the MQTT Client ID.
+* The ``<url>`` parameter is a string and it indicates the MQTT broker hostname.
+* The ``<port>`` parameter is an integer and it indicates the MQTT broker port.
+* The ``<sec_tag>`` parameter is an integer and it indicates the the credential of the security tag used for establish a secure connection.
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   #XMQTTEVT: <evt_type>,<result>
+
+* The ``<evt_type>`` value is an integer.
+  When ``0``, it indicates the acknowledgment of the connection request.
+* The ``<result>`` value is an integer and it can assume the following values:
+
+  * ``0`` - Connection succeded.
+  * *Negative Value* - Error code indicating the reason of the failure.
+
+Examples
+~~~~~~~~
+
+::
+
+   AT#XMQTTCON=1,"MyMQTT-Client-ID","mqtt.eclipse.org",1883
+   OK
+   #XMQTTEVT: 0,0
+
+::
+
+   AT#XMQTTCON=0
+   OK
+   #XMQTTEVT: 1,0
+
+Read command
+------------
+
+The read command is not supported.
+
+Test command
+------------
```

Comment:
The read command shows MQTT client information.

Syntax:
AT#XMQTTCON?
Response:
#XMQTTCON: <cid>,<username>,<password>,<url>,<port>[,<sec_tag>]]

Example:
AT#XMQTTCON?
#XMQTTCON: "MyMQTT-Client-ID","","","mqtt.eclipse.org",1883
OK

Reply:
@lats1980 Thanks for the note. Could you also specify the details of the <cid> value?

Reply:
@lats1980 Thanks for the input. Could you please specify also the details of the <cid> value?

Reply:
cid: String. MQTT Client ID.